diff -r -u gpsd-git/gpsd.hotplug.wrapper gpsd-git.new/gpsd.hotplug.wrapper
--- gpsd-git/gpsd.hotplug.wrapper	2011-06-12 11:00:02.649487747 +0100
+++ gpsd-git.new/gpsd.hotplug.wrapper	2011-06-14 15:02:20.716192680 +0100
@@ -12,6 +12,8 @@
   . /etc/sysconfig/gpsd
   GPSD_OPTIONS=$OPTIONS
   GPSD_SOCKET=$CONTROL_SOCKET
+elif [ -r /etc/rc.d/rc.gpsd.conf ]; then
+  . /etc/rc.d/rc.gpsd.conf
 fi
 
 if [ -n "$GPSD_OPTIONS" ]; then
