--- gpsd-git/SConstruct	2011-08-02 08:08:12.482517947 +0100
+++ gpsd-git.new/SConstruct	2011-08-03 11:08:57.604320691 +0100
@@ -148,13 +148,13 @@
 # set defaults from environment
 env['STRIP'] = "strip"
 env['CHRPATH'] = 'chrpath'
-for i in ["AR", "ARFLAGS", "CCFLAGS", "CFLAGS", "CC", "CXX", "CXXFLAGS", "STRIP", "CHRPATH", "LD", "TAR"]:
+for i in ["AR", "ARFLAGS", "CC", "CXX", "STRIP", "CHRPATH", "LD", "TAR"]:
     if os.environ.has_key(i):
         j = i
         if i == "LD":
             i = "SHLINK"
         env[j]=os.getenv(i)
-for flags in ["LDFLAGS", "CPPFLAGS"]:
+for flags in ["LDFLAGS", "CPPFLAGS", "CCFLAGS", "CFLAGS", "CXXFLAGS"]:
     if os.environ.has_key(flags):
         env.MergeFlags([os.getenv(flags)])
 
