diff -u -r gpsd-git//SConstruct gpsd-git.new//SConstruct
--- gpsd-git//SConstruct	2011-06-13 23:41:53.642101654 +0100
+++ gpsd-git.new//SConstruct	2011-06-13 23:33:03.186935002 +0100
@@ -184,8 +184,8 @@
                             -D_GNU_SOURCE'''))
 
 # Honor the specified installation prefix in link paths.
-env.Prepend(LIBPATH=[os.path.join(env['prefix'], 'lib')])
-env.Prepend(RPATH=[os.path.join(env['prefix'], 'lib')])
+env.Prepend(LIBPATH=[os.path.join(env['prefix'], env['libdir'])])
+env.Prepend(RPATH=[os.path.join(env['prefix'], env['libdir'])])
 
 # Tell generated binaries to look in the current directory for
 # shared libraries. Should be handled sanely by scons on all systems.
