diff -u -r gpsd-git//SConstruct gpsd-git.new//SConstruct
--- gpsd-git//SConstruct	2011-06-13 23:41:53.642101654 +0100
+++ gpsd-git.new//SConstruct	2011-06-13 23:33:03.186935002 +0100
@@ -850,6 +850,7 @@
     substmap = (
         ('@VERSION@', gpsd_version),
         ('@prefix@',  env['prefix']),
+        ('@libdir@',  env['libdir']),
         ('@PYTHON@',  sys.executable),
         )
     with open(str(source[0])) as sfp:
@@ -957,7 +958,7 @@
                     python_egg_info_install]
 
 pkgconfigdir = libdir + os.sep + 'pkgconfig'
-pc_install = [ env.Install(pkgconfigdir, "libgps.pc") ]
+pc_install = [ env.Install(pkgconfigdir, x) for x in ("libgps.pc", "libgpsd.pc") ]
 
 maninstall = []
 for manpage in base_manpages:
diff -u -r gpsd-git//libgps.pc.in gpsd-git.new//libgps.pc.in
--- gpsd-git//libgps.pc.in	2011-06-13 23:41:53.642101654 +0100
+++ gpsd-git.new//libgps.pc.in	2011-06-13 23:39:14.170666019 +0100
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=${exec_prefix}@libdir@
 includedir=${prefix}/include
 
 Name: GPSD
diff -u -r gpsd-git//libgpsd.pc.in gpsd-git.new//libgpsd.pc.in
--- gpsd-git//libgpsd.pc.in	2011-06-13 23:41:53.642101654 +0100
+++ gpsd-git.new//libgpsd.pc.in	2011-06-13 23:39:31.758273331 +0100
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=${exec_prefix}@libdir@
 includedir=${prefix}/include
 
 Name: libgpsd
